lib/gs-plugin-types.h

```diff
@@ -320,6 +320,7 @@ typedef enum {
 	GS_PLUGIN_ACTION_PURCHASE,
 	GS_PLUGIN_ACTION_GET_COPYABLE,
 	GS_PLUGIN_ACTION_COPY,
+	GS_PLUGIN_ACTION_OS_GET_COPYABLE,
```

Comment:
Needs to be added to the documentation above.

Reply:
I assume it’s not a problem that adding it here will shift the values of the following elements in the enum? i.e. The numeric values of the elements are not part of any API?

Reply:
It's technically public API, so I've fixed the ordering locally.

In practice, it probably won't matter since this would only be an API break downstream at this point and we'll keep the library and all dependents synchronized. The only dependents are plugins and all the ones we use, as far as I know, are in this source tree. It's possible third-party plugins exist but I don't know.